internal/appsec/appsec.go

```diff
@@ -199,6 +199,10 @@ func (a *appsec) stop() {
 		a.wafHandle.Close()
 		a.wafHandle = nil
 	}
+
+	// Reset rules edits received from the remote configuration
+	a.cfg.RulesManager, _ = config.NewRulesManager(nil)
```

Comment:
Silently ignoring the error that might be returned here? If we can guarantee there cannot be an error here, it'd be nice to have a comment explaining why that is safe to assume...